.editorconfig

```diff
@@ -0,0 +1,43 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,py}]
+charset = utf-8
+
+# 4 space indentation
+[*.py]
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+# Tab indentation (no size specified)
+[Makefile]
+indent_style = tab
+
+[*.js]
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
+
+[*.html]
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 1
```

Comment:
The currently used coding style is 4 spaces per level. I'd stick with that unless we explicitly decide to change it.